README.adoc

```diff
@@ -1,4 +1,37 @@
 = Epsilon-Telemetry-Server
 
+= Deploying to the server
+
+First you want to ssh into the IP
+`ssh root@138.197.206.227`
+Refer to current manager for password
+`cd Epsilon-Telemetry-Server`
+`git pull` and/or checkout to branch you want to deploy
+
+== For database changes or server side changes
+`cd server`
+
+To remake the database
+`sudo -u postgres dropdb epsilontelemetrydb`
+`sudo -u postgres createdb epsilontelemetrydb`
+`sudo -u postgres psql epsilontelemetrydb < migrate.psql`
+
+After integrating the changes and/or remaking the database, you want to kill the current server and start it again
+`pgrep node` finds the server process
+`kill server_process`
+`npm run start` to ensure server is killed
+If so, then ctrl-c to end that process
```

Comment:
```
`npm run start` to ensure server is killed
If so, then ctrl-c to end that process
```
Don't really need this step, since we know it was killed by pgrep, and they can use pgrep to check again.